src/nccl_ofi_net.c

```diff
@@ -696,7 +696,7 @@ int nccl_ofi_init_connection(struct fi_info *info, struct fid_domain *domain,
 				NCCL_OFI_WARN("GDR support reported to NCCL but then couldn't be configured on an endpoint.  Cannot continue.");
 				goto error;
 			} else {
-				NCCL_OFI_TRACE(NCCL_INIT | NCCL_NET, "Could not disable CUDA API usage for HMEM, disabling GDR");
+				NCCL_OFI_WARN("Could not disable CUDA API usage for HMEM, disabling GDR");
```

Comment:
I don't think this should be a warn.  I think info is right.  Otherwise, this warning is going to pop on every run on the P3dn and G* platforms.